### PR TITLE
Feat/public service list

### PR DESCRIPTION
**Pull request by andskur**

### CODE REVIEW

**Review by dmitriyselivanov — APPROVED** [review]

File: services_list.go [hunk 1]
```diff
@@ -186,20 +186,20 @@ func (l *ServicesList) HealthChecksLoop() {
 
 	for {
 		select {
-		case <-l.stop:
-			logger.Log().Warn("stop healthchecks loop")
+		case <-l.Stop:
+			logger.Log().Warn("Stop healthchecks loop")
```
Comment (maxrev17) [inline services_list.go:190]: Really picky - but conventionally don't capitalise the start of a log entry? (surprised linter doesn't pick this up)

**Review by maxrev17 — APPROVED** [review]
Looks fine, guessing everything is now exported so can be accessed by calling code - is there any risk associated with this?

**Comment by andskur** [comment]
> Looks fine, guessing everything is now exported so can be accessed by calling code - is there any risk associated with this?

we should be more carefully =)

File: services_list.go (hunk 1 above)
Comment (andskur) [reply services_list.go:190]: oh, good catch! thanks!

**Review by nanevardanyan — APPROVED** [review]

**Review by Misnaged — APPROVED** [review]

**Pull request merged**